.github/workflows/mingw.yml

```diff
@@ -221,20 +221,20 @@ jobs:
 
 
   cross-llvm-mingw:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
curious why we had specified the version in the first place.

Reply:
so it setup-python downloads a Python that matches https://hub.docker.com/r/mstorsjo/llvm-mingw/tags/ apparently, which is also why we get GLIBC linker errors now

Reply:
We need to port away from setup-python I think

Reply:
> We need to port away from setup-python I think and/or ask llvm-mingw if they can maybe update their base image

Not sure what we can use here? Maybe there's some apt repository to get the correct version?

Reply:
I went with the standard deadsnakes PPA